client/index.css

```diff
@@ -199,3 +199,112 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    width: 345px;
+    height: auto;
+    margin: 40px auto 0 auto;
+}
+#settings #icon-username {
+    margin: 18px 0 0 100px;
+    overflow: auto;
+}
+#icon-username h2 {
+    margin-left: 120px;
+}
+#settings .icon {
+    width: 100px;
+    height: 150px;
+    border: 1px solid black;
+    float: left;
+    margin-top: -45px;
+}
+.icon img {
+    height: 100px;
+    width: 98px;
+}
+.icon span {
+    padding: 4px 10px;
+    border-top: 1px solid black;
+    cursor: pointer;
+    display: block;
+}
+#settings #form {
+    margin-top: 50px;
+    width: 400px;
+}
+#form select {
+    width: 174px;
+    display: inline-block;
+    padding: 0 0 0 9px;
```

Comment:
[I don't see a difference](https://www.youtube.com/watch?v=hSycqmINnM0&feature=youtu.be). Why do you need the padding?